app/components/take-photo.js

```diff
@@ -108,7 +108,8 @@ export default Ember.Component.extend({
           photo,
           video
         });
-        if (typeof MediaStreamTrack === 'undefined' || MediaStreamTrack.getSources === 'undefined') {
+        if (typeof MediaStreamTrack === 'undefined' ||
```

Comment:
ESLint is complaining about this:
 111:55  error  '||' should be placed at the beginning of the line  operator-linebreak
so if you could move the || to the beginning of the next line, it should address the failing test.